.github/workflows/ci.yml

```diff
@@ -12,8 +12,12 @@ env:
 jobs:
   build:
     runs-on: ubuntu-latest
-    name: Ruby ${{ matrix.ruby }}
+    name: Ruby ${{ matrix.ruby }} (Rack ~> ${{ matrix.rack }})
     services:
+      redis:
```

Comment:
Since there was only one version of Redis tested, let’s use the official Redis image as a service.